base/applications/rapps/lang/sk-SK.rc

```diff
@@ -167,19 +167,19 @@ END
 STRINGTABLE
 BEGIN
     IDS_CAT_AUDIO "Audio"
-    IDS_CAT_DEVEL "Vývojárske" // Development, vývoj??
-    IDS_CAT_DRIVERS "Ovládače" // Drivers
-    IDS_CAT_EDU "Vzdelávacie" // Edutainment
-    IDS_CAT_ENGINEER "Inžinierske" // Engineering, konštrukčné??
-    IDS_CAT_FINANCE "Finančné" // Finance, peňažníctvo??
-    IDS_CAT_GAMES "Hry & zábava" // Games & Fun, zábavné??
-    IDS_CAT_GRAPHICS "Grafické" // Graphics
```

Comment:
@Venca24 could you double-check the old commented suggestions and tell us whether the currently-used ones are indeed better?

Reply:
@HBelusca Generally yes, the comments were just noting which was the original name of the category or possible other options for the name, but they are not so much better.

But it is noteworthy that the style of the names is different: whereas in other languages (including Czech) the names of the categories are nouns, those in this Slovak translation are adjectives saying "of which purpose is the software".

Reply:
Feel free to improve on that (later) if you can come up with good suggestions. For now I want to avoid touching anything but comments and accels in this PR. I do also have some other TODOs for myself in rapps. But I didn't want to mix them into here, to not make it too hard to review.